### ISSUE TITLE
Getting 0 loss and MAPE for model with dataset downloaded and in the correct configuration.

### ISSUE THREAD

**Issue by sneedfeednseed66** [Mengxue12/RouteNet-E_pytorch#1]
Hi, 

First off I want to thank you for going through the time to port this from tensorflow to pytorch. The original Routenet builds were horrible and have been an obstacle to validating or improving on the paper's original work.

That being said I have an issue getting the training to work as i keep getting 0 error. I looked through the code and dont know if I am running a pre-trained model with same test and train data or if I totally forgot to set up the data correctly. 

Regardless, how did you set it up when running this main.py?

**Comment by rohinkalra21** [comment]
> Hi,
> 
> First off I want to thank you for going through the time to port this from tensorflow to pytorch. The original Routenet builds were horrible and have been an obstacle to validating or improving on the paper's original work.
> 
> That being said I have an issue getting the training to work as i keep getting 0 error. I looked through the code and dont know if I am running a pre-trained model with same test and train data or if I totally forgot to set up the data correctly.
> 
> Regardless, how did you set it up when running this main.py?

I also would like to know this.
